integration/specs/InternalOverlay/internalOverlay-1.spec.js

```diff
@@ -15,18 +14,18 @@ describe('InternalOverlay with default position resolver', () => {
     });
 
     it('should exist after one click on trigger and destroy on second click', () => {
-        const button = new PageInternalOverlayButton(BUTTON);
         const internalOverlay = new PageInternalOverlay(OVERLAY);
-        button.click();
+        const triggerButton = $(BUTTON);
+        triggerButton.click();
         expect(internalOverlay.exists()).toBe(true);
-        button.click();
+        triggerButton.click();
         expect(internalOverlay.exists()).toBe(false);
     });
 
     it('should not exists after click outside of element', () => {
-        const button = new PageInternalOverlayButton(BUTTON);
         const internalOverlay = new PageInternalOverlay(OVERLAY);
-        button.click();
+        const triggerButton = $(BUTTON);
+        triggerButton.click();
         expect(internalOverlay.exists()).toBe(true);
         $('[data-preview="InternalOverlay"]').click();
```

Comment:
It is not okay to use an element outside the example for integration testing because it makes them vulnerable.

I think the best solution is to add an id to the Container of the example, this way you can use it as a trigger for the OutsideClick.